templates/include/glean/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Version 1
 
+- Added optional `session_id` and optional `session_count` client info field. See [Bug 1862955](https://bugzilla.mozilla.org/show_bug.cgi?id=1862955).
```

Comment:
Side note: looks like  https://bugzilla.mozilla.org/show_bug.cgi?id=1751739 is missing from this history.

Reply:
Fixed https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/795/commits/4e2238736edf229b1073d1def879703b6d7a6adf